bip-musig2.mediawiki

```diff
@@ -347,9 +347,10 @@ Algorithm ''ApplyTweak(keygen_ctx, tweak, is_xonly_t)'':
 === Nonce Generation ===
 
 <div>
-Algorithm ''NonceGen(sk, aggpk, m, extra_in)'':
+Algorithm ''NonceGen(sk, pk, aggpk, m, extra_in)'':
 * Inputs:
 ** The secret signing key ''sk'': a 32-byte array (optional argument)
+** The plain public key ''pk'': a 33-byte array
```

Comment:
Nit: Does it make sense for mandatory argument to be first?

Reply:
Is there a convention to do this? I went with the convention where usually the secret key is stated before the public key.